closure/bin/build/closurebuilder.py

```diff
@@ -47,7 +47,7 @@
   ^(mc\.          # Must be a property of the monetate common namespace. Capture group 1.
   [a-zA-Z]+\.     # Must have it's own module namespace. Capture group 1.
   [A-Z0-9_]+)     # Must follow the constant format. Capture group 1.
-  =(true|false)$  # Must assign a boolean. Capture group 2.
+  =(.*)$          # Must assign a value. Capture group 2.
```

Comment:
Capturing anything seems a bit "dangerous" since now there will be no true validation on the value of the feature flag. Could potentially put values in there that aren't valid for the flag being set and nothing will be spit out as an error.